cargo-apk/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+- **Breaking**: Default `target_sdk_version` to `29` or lower for more consistent
+  interaction with Android backwards compatibility and tighter rules:
```

Comment:
Not a fan of this wording, will see if I can come up with something better after a night's
 sleep.

Reply:
Updated. @msiglreith what do you think?